cas/ctg_store.h

```diff
@@ -137,7 +137,12 @@ enum m0_cas_state_format_version {
 enum {
 	/**
 	 * Every key and value is stored with an individual 64-bit header.
-	 * Currently header contains key/value length in bytes.
+	 * For the keys in all cas btrees, key header contains key length
+	 * in bytes.
+	 * For CT_META, CT_BTREE, CT_DEAD_INDEX type of cas btrees, value header
```

Comment:
I am not able to find if the version field is added for CT_DEAD_INDEX tree. Can you please check it again?

Reply:
Updated comment.